docs/specs/IDE-execution.md

```diff
@@ -36,7 +36,16 @@ If multiple replicas are used (created via `ExecutableReplicaSet` objects), all
 
 ## IDE session endpoint requests
 
-The IDE session endpoint is expected to support the following requests, delivered via HTTP protocol, and using JSON as the payload format:
+The IDE session endpoint accepts requests delivered via HTTP protocol. All request use JSON as the request/response payload format, as appropriate per request type.
+
+### Mandatory request headers
+
+Every request must contain the following headers:
+
+| Header | Value | Description |
+| --- | --- | ------ |
+| `Authorization` | `Bearer <security token>` | Security (bearer) token, as read from `DEBUG_SESSION_TOKEN` environment variable. Authenticates the DCP instance to IDE session endpoint. |
+| `Microsoft-Developer-DCP-Instance-ID` | `<unique ID>` | A unique ID generated by DCP for every DCP program execution. Can be used by the IDE to distinguish between multiple DCP instances running in parallel. <br/> If `DCP_INSTANCE_ID_PREFIX` environment variable is set, the contents of this variable will be prepended to the unique instance ID DCP generates. |
```

Comment:
Is there a separator between the DCP_INSTANCE_ID_PREFIX and the unique instance ID?

Reply:
No, but the prefix is not parsed/interpreted by DCP in any way, so if you need to have some separator between the ID that DCP generates and the prefix, you can add that at the end of the `DCP_INSTANCE_ID_PREFIX` value